resources/css/app.css

```diff
@@ -1,3 +1,4 @@
 @import "tailwindcss" theme(static);
-@source "../views/";
-@source "../../app/";
+@source '../../storage/framework/views/*.php';
```

Comment:
This makes sense but it means things will break if a user [changes where their compiled view files are located](https://roots.io/acorn/docs/directory-structure/). If possible, we should try to generate this path instead of hard coding it.